WordPress/Resources/en.lproj/Localizable.strings

```diff
@@ -9485,6 +9615,9 @@
 /* This text is used when the user is configuring the iOS widget to suggest them to select the site to configure the widget for */
 "ios-widget.gpCwrM" = "Select Site";
 
+/* This text is required by Apple to be part of the translations but is not shown to the users. You can use the English version verbatim without translation for this entry. */
+"ios-widget.tVvJ9c" = "Select Site Intent";
```

Comment:
👆 just a note here to confirm the automation added this entry. Not that there were doubts about it, but I didn't take it into account when I originally proposed this, which would have resulted in unnecessary noise in the Git log. See https://github.com/wordpress-mobile/WordPress-iOS/pull/18254#discussion_r840300107